OPCB/functions/missions/Resupply.sqf

```diff
@@ -4,36 +4,40 @@ _taskcomp = selectRandom ["fob1","fob2","fob3","fob4","fob5"];
 _officergroup = createGroup [west,true];
 
 _current_task = _base getPos[random 600,random 360];
-[_current_tasknumber ,west,["Nearby friendly forces have requested our help. You have to bring supplies to them. You have abou 15 minutes left before the attack begins.","Resupply"], _current_task,"ASSIGNED",10,true,true,"rearm",true] call BIS_fnc_setTask;
+[_current_tasknumber ,west,
+["Nearby friendly forces have requested our help. You have to bring supplies to them. You have abou 15 minutes left before the attack begins.",
+"Resupply"], _current_task,"ASSIGNED",10,true,true,"rearm",true] call BIS_fnc_setTask;
+
 _officer = _officergroup createUnit ["rhsusf_usmc_marpat_d_officer", _base, [], 2, "NONE"];
 _officer disableAI "MOVE";
 
-_container = "C_IDAP_supplyCrate_F" createVehicle (getMarkerPos "object_dropoff");
+_container = "C_IDAP_supplyCrate_F" createVehicle (getPos dropoffpoint);
 [_container, 5] call ace_cargo_fnc_setSize;
 clearItemCargoGlobal _container;
 _guardpos = getPos _officer;
 _comp = [_taskcomp,_guardpos, [0,0,0], random 360, true, true ] call LARs_fnc_spawnComp;
 
-_markpos1 = _guardpos getPos[50,random 360];
-_markpos2 = _guardpos getPos[50,random 360];
-_defender1 = [_markpos1, west,["rhsusf_army_ocp_rifleman_m590","rhsusf_army_ocp_rifleman_m16","rhsusf_army_ocp_riflemanat","rhsusf_army_ocp_riflemanl","rhsusf_army_ocp_rifleman_m4","rhsusf_army_ocp_rifleman_1stcav","rhsusf_army_ocp_rifleman_10th","rhsusf_army_ocp_rifleman_m16"]] call BIS_fnc_spawnGroup;
-_defender2 = [_markpos2, west,["rhsusf_army_ocp_rifleman_m590","rhsusf_army_ocp_rifleman_m16","rhsusf_army_ocp_riflemanat","rhsusf_army_ocp_riflemanl","rhsusf_army_ocp_rifleman_m4","rhsusf_army_ocp_rifleman_1stcav","rhsusf_army_ocp_rifleman_10th","rhsusf_army_ocp_rifleman_m16"]] call BIS_fnc_spawnGroup;
-
-[_defender1, _guardpos, 100] call bis_fnc_taskPatrol;
-[_defender2, _guardpos, 100] call bis_fnc_taskPatrol;
-
-[[_defender2,_defender1]] call CHAB_fnc_serverGroups;
+_defenders = [];
+for "_i" from 0 to 2 do { 
+	_markpos = _guardpos getPos[50,random 360];
+ 	_defender = [_markpos, west,["rhsusf_army_ocp_rifleman_m590","rhsusf_army_ocp_rifleman_m16","rhsusf_army_ocp_riflemanat","rhsusf_army_ocp_riflemanl","rhsusf_army_ocp_rifleman_m4","rhsusf_army_ocp_rifleman_1stcav","rhsusf_army_ocp_rifleman_10th","rhsusf_army_ocp_rifleman_m16"]] call BIS_fnc_spawnGroup;
```

Comment:
These classnames could also be externally parametrised, for sharing with other future tasks and improving portability in case we want to have a different BLUFOR faction.